agent/dns.go

```diff
@@ -1497,6 +1658,7 @@ func (d *DNSServer) serviceSRVRecords(cfg *dnsConfig, dc string, nodes structs.C
 		}
 		handled[tuple] = struct{}{}
 
+<<<<<<< HEAD
```

Comment:
Looks like a git rebase conflict needs handling here.

Reply:
@mkeeler That's right. I've pushed the fixed version.